components/src/molecules/Tabs/Tabs.stories.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react'
+import { useArgs } from '@storybook/preview-api'
+import { Tabs as TabComponent } from '.'
+import type { Meta, StoryObj } from '@storybook/react'
+
+const meta: Meta<typeof TabComponent> = {
+  title: 'Library/Molecules/Tabs',
+  component: TabComponent,
+
+  argTypes: {
+    tabs: {
+      control: {
+        type: 'array',
+      },
+    },
+  },
+}
+
+export default meta
+
+type Story = StoryObj<typeof TabComponent>
+
+export const Tabs: Story = {
+  args: {
+    tabs: [
+      {
+        text: 'Setup',
+        isActive: false,
```

Comment:
when the app displays tabs, setup is selected so settings isActive: true would be designer-friendly. 

```suggestion
        isActive: true,
```